src/modules.d.ts

```diff
@@ -89,6 +89,8 @@ declare namespace Electron {
         getOwnerBrowserWindow: () => BrowserWindow | void;
         mainFrameRoutingId: number;
         session: Session;
+        //DELETE IN v13
+        sendToFrame(frameId: number, channel: string, ...args: any[]): void;
```

Comment:
Is this supposed to be removed?

Reply:
I'll do it in a follow up so that the merge itself is clean